packages/sprotty-library/package.json

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "sprotty-library",
+    "version": "0.0.1",
+    "description": "Views and elements library for Sprotty",
+    "license": "(EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0)",
+    "keywords": [
+        "eclipse",
+        "graphics",
+        "diagram",
+        "modeling",
+        "visualization",
+        "svg"
+    ],
+    "homepage": "https://github.com/eclipse-sprotty/sprotty",
+    "bugs": "https://github.com/eclipse-sprotty/sprotty/issues",
+    "author": {
+        "name": "Eclipse Sprotty"
+    },
+    "peerDependencies": {
+        "inversify": "~6.0.2"
+    },
+    "dependencies": {
+        "sprotty": "^1.1.0",
+        "sprotty-protocol": "^1.1.0"
```

Comment:
`inversify` and `sprotty-protocol` are transitive dependencies of `sprotty`, so they can be removed here.